### PR TITLE
Update: dependencys.

### PR DESCRIPTION
**Pull request by frank-dspeed**
## Commit Message Guidelines
```
Header
Blank Line
Body
Blank Line
Footer

The header should look like:
<type>(<scope>): <subject>

The body should have any necessary detailed info about the commit:
An example, references as to where this idea came from, etc.

The footer should have all the issues tagged:
Fixes #123, Fixes #456

So a commit should look like:
feat(users): Add new Yahoo authentication

Yahoo authentication idea proposed by @codydaig
Example implementation in file.js

Fixes #82
```

* Types: 
  * feat - Features, Enhancements, and overall Improvements
  * fix - Fixes, Bugs, HotFixs, etc...
  * doc - Changes to the Documentation and doesn't actually touch any code.
* Scope:
  * The scope should be where the change took place.
  * Examples: users, core, config, articles
* Subject:
  * The subject line should be clear and concise as to what is being accomplished in the commit.
* General Rules:
  * No Line in the Commit message can be longer than 80 characters.
* Reference: [Angular Conventions](https://github.com/ajoslin/conventional-changelog/blob/master/conventions/angular.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,26 +71,26 @@
...
     "winston": "^2.2.0",
     "wiredep": "~4.0.0"
   },
   "devDependencies": {
     "coveralls": "~2.11.6",
     "del": "^2.2.2",
-    "eslint": "~2.2.0",
-    "eslint-config-airbnb": "~6.0.2",
+    "eslint": "~3.9.1",
+    "eslint-config-airbnb": "~13.0.0",
```
Comment (simison) [inline package.json]: AFAIK, this won't work without some changes at the `gulpfile.js` first.

**Comment by simison** [comment]
@frank-dspeed did you test run this?

**Comment by frank-dspeed** [comment]
@simison  nope did not and why does this project not have automated tests should i add some?

**Comment by mleanos** [comment]
@frank-dspeed I think Mikael was referring to whether or not you ran the tests locally, and *perhaps* did a quick smoke check by manual testing the app. As you can see the linting tasks is throwing a lot of errors.

File: package.json (hunk 1 above)
Comment (mleanos) [inline package.json]: This is a big bump and causing a lot of linting errors. Any insight into why? Have the linting rules changed that much?

Perhaps new rules have been added with defaults that we are violating?

**Comment by mleanos** [comment]
@simison Do you think it makes sense to force the build to fail when linting errors occur? I'm a bit torn on that because the linting rules are more of styling, and our test suites should reflect the *working state* of the framework.

**Comment by simison** [comment]
@mleanos personally I would fail the build on linting errors, as it's otherwise harder to spot them — and why wouldn't you want to ask linting errors to be corrected anyway...

File: package.json (hunk 1 above)
Comment (simison) [inline package.json]: This comment is relevant here: https://github.com/meanjs/mean/issues/1569#issuecomment-254018691 (@sujeethk)

**Comment by mleanos** [comment]
Ok. I think we're in agreement there then. It's definitely been a pain to find linting errors after a merge.

@frank-dspeed Can you look into fixing the linting errors with this PR & figuring out a way to have the builds fail if the linting tasks fail?

**Comment by frank-dspeed** [comment]
yes i am working on this

**Comment by simison** [comment]
@frank-dspeed I recon the linting bit could be its own PR and this PR can upgrade everything else except those eslint bits?

**Comment by frank-dspeed** [comment]
@simison done

File: package.json [hunk 2]
```diff
@@ -71,26 +71,26 @@
...
+    "validator": "~6.1.0",
     "winston": "^2.2.0",
     "wiredep": "~4.0.0"
   },
   "devDependencies": {
     "coveralls": "~2.11.6",
     "del": "^2.2.2",
     "eslint": "~2.2.0",
-    "eslint-config-airbnb": "~6.0.2",
+    "eslint-config-airbnb": "~13.0.0",
```
Comment (simison) [inline package.json]: @frank-dspeed this, too.

Comment (frank-dspeed) [inline package.json]: oh even that is incompatible damn damn much work :D

**Comment by mleanos** [comment]
@frank-dspeed I just re-ran the build to see if the same errors are thrown. It looks like it could be an issue for Node v4. What was updated in this PR since my review?

**Comment by mleanos** [comment]
@frank-dspeed The build is still failing. It appears that eslint `v2.2.1` isn't a valid version. Can you change it back to `2.2.0`?

https://travis-ci.org/meanjs/mean/jobs/176696521#L883

**Comment by frank-dspeed** [comment]
@mleanos yes working on that simply wait till the build passes will ping you then.

**Comment by mleanos** [comment]
@frank-dspeed Friendly reminder :)

**Comment by simison** [comment]
@mleanos @lirantal this can be closed as dependencies are already newer in master.

**Comment by lirantal** [comment]
Agreed.

**Pull request closed without merging**